src/views/SystemIntake/ContactDetails/index.tsx

```diff
@@ -153,14 +153,14 @@ const ContactDetails = ({ formikProps }: ContactDetailsProps) => {
 
           <Field
             as={RadioField}
-            checked={values.isso.isPresent === true}
+            checked={values.isso.isPresent}
```

Comment:
We actually need the `=== true` here because it is a controlled component.
https://fb.me/react-controlled-components

You can see the error in the dev tools when you load up the page and click `Yes`